test/jasmine/tests/frame_api_test.js

```diff
@@ -0,0 +1,205 @@
+var Plotly = require('@lib/index');
+
+var createGraphDiv = require('../assets/create_graph_div');
+var destroyGraphDiv = require('../assets/destroy_graph_div');
+
+// A helper function so that failed tests don't simply stall:
+function fail(done) {
```

Comment:
That's a great idea. :clap: 

[Note to self] Maybe move this to `test/jasmine/assets/` .